### DISCUSSION TITLE
How to access a restricted website due to nonpayment (university of zambia) institute of distance learning

### DISCUSSION THREAD

**Discussion by Givenshimwaka** [community/community#73037] [GitHub Education]
### Hi everyone,

  Good day, I am a new member of the great GitHub.
       being a drop-out undergraduate in the school of business studies in particular business administration, I need help from you to help me access the school Moodle where I will be able to access the school entrance at a premium level. help me with the Chrome extension to the school's website.

   school website: https://online.unza.zm/users/login

**Comment by oussamaef06** [comment]
I understand that you are looking for assistance with accessing your school's Moodle platform. However, it's important to note that accessing any website, especially educational platforms, should be done through legitimate means and in compliance with your school's policies and procedures.

If you are experiencing difficulties accessing your school's Moodle platform, I would recommend taking the following steps:

Contact Your School's IT Support: Reach out to your school's IT support or helpdesk. They can provide guidance and assistance with any technical issues you may be facing when trying to access the Moodle platform.

Verify Your Credentials: Ensure that you are using the correct username and password provided by your school to access the platform. If you are unsure about your credentials, contact your school's administration.

Browser Compatibility: Make sure you are using a compatible web browser. Moodle is usually compatible with popular browsers like Chrome, Firefox, and Safari. Ensure that your browser is up to date.

Clear Browser Cache: Sometimes, browser cache can cause issues with logging in. Try clearing your browser's cache and cookies, and then attempt to log in again.

Check for URL Accuracy: Double-check the URL you provided. Ensure it's the correct URL for your school's Moodle platform.

Mobile App: Some schools provide dedicated mobile apps for accessing Moodle. Check if your school has a mobile app available and try using it.

Network Connection: Ensure that you have a stable and secure network connection. Unstable or insecure networks can lead to login issues.

Read School Policies: Make sure you are following your school's policies and guidelines for accessing their online resources. Attempting to bypass any security measures can have serious consequences.

If you are facing specific issues with a Chrome extension, I would advise against using any extensions or tools that are not provided or recommended by your school's IT department, as this may violate your school's policies and potentially lead to academic or legal consequences.

It's essential to approach this situation in a legitimate and ethical manner and seek assistance from the appropriate channels to resolve any access issues.

**Comment by samus-aran** [comment]
Hi 👋 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as Moodle. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
